pareto/utilities/process_data.py

```diff
@@ -0,0 +1,838 @@
+    )
+
+    for t in strategic_model.s_T:
+        strategic_model.p_beta_TotalPW[t] = sum(
+            strategic_model.p_beta_Production[p, t]
+            + strategic_model.p_beta_Flowback[p, t]
+            for p in strategic_model.s_P
+        )
+
+        strategic_model.p_sigma_MaxPWCapacity[t] = (
+            sum(
+                strategic_model.p_gamma_Completions[p, t]
+                for p in strategic_model.s_P  # Completions Demand
+            )
+            + sum(
+                strategic_model.p_omega_EvaporationRate
+                for s in strategic_model.s_S  # Storage Evaporation
+            )
+            + sum(
+                strategic_model.p_sigma_BeneficialReuse[o, t]
+                if strategic_model.p_sigma_BeneficialReuse[o, t].value >= 0
+                else strategic_model.p_M_Flow
+                for o in strategic_model.s_O  # Beneficial Reuse (if user does not specific value, p_sigma_BeneficialReuse is -1)
+                # In this case, Beneficial Reuse at this site has no limit (big M parameter)
+            )
+            + sum(
+                strategic_model.p_sigma_Storage[s]
+                + _get_max_value_for_parameter(strategic_model.p_delta_Storage)
+                for s in strategic_model.s_S  # Storage
+            )
+            + sum(
+                (
+                    strategic_model.p_sigma_Disposal[k]
+                    + _get_max_value_for_parameter(strategic_model.p_delta_Disposal)
+                )
+                * strategic_model.p_epsilon_DisposalOperatingCapacity[k, t]
+                for k in strategic_model.s_K  # (Initial Disposal + max disposal) * operating capacity
+            )
+            + sum(
+                max(
+                    [
+                        (
+                            strategic_model.p_sigma_Treatment[r, wt].value
+                            + max(
+                                [
+                                    strategic_model.p_delta_Treatment[wt, j].value
+                                    for j in strategic_model.s_J
+                                ]
+                            )
+                        )
+                        * (1 - strategic_model.p_epsilon_Treatment[r, wt].value)
+                        for wt in strategic_model.s_WT
+                    ]
+                )
+                for r in strategic_model.s_R
+            )  # Treatment residual: for each treatment site - treatment technology that yields the
+            # maximum value for: (initial treatment + max treatment expansion) * treatment efficiency
+        )
+        # Note: if completions pad is outside system, demand is not required to be met
+        strategic_model.p_gamma_TimePeriodDemand[t] = sum(
+            strategic_model.p_gamma_Completions[cp, t]
+            * (1 - strategic_model.p_chi_OutsideCompletionsPad[cp])
```

Comment:
```suggestion
            if strategic_model.p_chi_OutsideCompletionsPad[cp] == 0 else 0
```
I think this is a more straightforward way to write this logic.